src/filesystem/drive/inner.rs

```diff
@@ -20,7 +23,7 @@ pub(crate) struct InnerDrive {
     /// This is the filesystem-wide vector clock used for tracking what permanent IDs are present
     /// within the filesystem as well as access control related changes. Changes to individual
     /// node's or their entry specific data should make use of the [`Node`]'s vector clock instead.
-    vector_clock: VectorClock,
+    vector_clock: VectorClockFilesystem,
     root_pid: PermanentId,
```

Comment:
I suspect we'll want to also store the current actor's vector clock either here or in the outer drive (most likely here) along side the FS clock